src/batch/src/executor/group_top_n.rs

```diff
@@ -185,7 +192,7 @@ impl<K: HashKey> GroupTopNExecutor<K> {
             {
                 let heap = groups
                     .entry(key)
-                    .or_insert_with(|| TopNHeap::new(self.limit, self.offset));
+                    .or_insert_with(|| TopNHeap::new(self.limit, self.offset, self.with_ties));
                 heap.push(HeapElem {
```

Comment:
(off the topic) The struct `HeapElem` keeps an `Arc` to the whole chunk, which may cause at most `CHUNK_SIZE`-times space amplifying. This is a little problem for TopN since the amount of data is bounded by `N`, but might be more severe for group TopN.

Reply:
If others agree with me, let's create another issue for it.

Reply:
It seems that we've already stored the encoded row, so we definitely can remove the chunk field. Currently, we use the `chunk` field in `append_one_row_ref`. If it can accept an encoded row directly, then we can remove the `chunk` here.